### DISCUSSION TITLE
Consent banner showing repeatedly

### DISCUSSION THREAD

**Discussion by maximobesteiro** [orestbida/cookieconsent#566] [Q&A]
Hello all! I'm getting reports from some of our platform's users about seeing the consent banner popping up after a page refresh when they had already provided consent.
This seems to be only happening on Chrome.
It looks like the cc_cookie is being removed for some reason that we cannot figure out.

Anyone else came across this?
Thanks!

**Comment by ahorovit** [comment]
I'm also seeing this issue. Any advice would be much appreciated!

**Comment by orestbida** [comment]
@maximobesteiro, please provide a demo!

**Reply by maximobesteiro** [reply]
Extremely hard to repro, that's why I posted a message here to see if anyone else had come across this in an attempt to understand if this was a problem with the library itself or an implementation issue on our side. Will update this thread in case we get to the root cause. Thanks for chiming in @orestbida 🙌🏼
